hs_core/models.py

```diff
@@ -1510,6 +1531,30 @@ def delete_all_elements(self):
         self.sources.all().delete()
         self.relations.all().delete()
 
+    def copy_all_elements_to(self, tgt_res):
```

Comment:
it not a big deal but logically i think a utility that copies elements into the current makes more sense to see. It makes more sense for a method to modify its own resource, rather than another.